src/Nethermind/Directory.Build.props

```diff
@@ -1,5 +1,12 @@
 <Project>
 
+  <PropertyGroup Condition="'$(CI)' == 'true'">
+    <!-- Required due to an issue with transitive dependencies of Colorful.Console -->
+    <!-- See: https://github.com/tomakita/Colorful.Console/pull/94 -->
+    <NoWarn>NU1605</NoWarn>
```

Comment:
`Colorful.Console` does not officially support .NET 8 which causes issues when specifying runtimes. I'm not sure if the issue just does not show up in "normal" builds but is still there, or is it only due to the usage of specific runtimes.

Microsoft's docs are quite poor in this case and the proposed workaround just does not work: https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605#example-3

Reply:
Yet another reason to deprecate Nethermind.Cli.